Makefile.am

```diff
@@ -115,7 +115,9 @@ po-pull:
 	TEMP_DIR=$$(mktemp --tmpdir -d anaconda-localization-XXXXXXXXXX) && \
 	git clone --depth 1 -b $(GIT_L10N_BRANCH) -- $(L10N_REPOSITORY) $$TEMP_DIR && \
 	cp $$TEMP_DIR/$(L10N_DIR)/*.po $(srcdir)/po/ && \
-	rm -rf $$TEMP_DIR
+	rm -rf $$TEMP_DIR && \
+	sed -i 's/X-Generator:\ Zanata 3\.5\.1//' $(srcdir)/po/kn.po && \
+	find $(srcdir)/po -type f -exec sed -ri '/^\"Plural-Forms:/ s/;(\\n")$$/\1/' {} +
```

Comment:
`Plural-Forms` should have a semicolon at the end. Please verify that plurals actually work with this change.

Reply:
To expand on that: As I understand it, you consume the resulting translations with cockpit, so this tweaks the translation for the cockpit tech, which is based on Javascript. However, the GTK and text modes may work differently, as they use actual gettext binaries (`.so`?) via Python's FFI or some such. You are making a single translation file (IMO correct), so the changes for one tech stack may break the other one. That's my concern.

Reply:
Your concern makes sense. I can think of three ways to address this:
* Instead of symlinking po/ with ui/webui/po create the latter with this sed line to be applied extra.
* Or verify if the missing semicolon actually does not create any problem, in this case we can just leave it?
* Or find exactly the project that creates the bug linked in the commit message, and address the issue upstream, wait for a release and have it fixed for real

Reply:
I'd say the "verify if does not cause problems" is the right first choice here - I think the parsing may just survive that bit. Or not.

Can you do that, or should I help with this?

Reply:
I guess I can build the RPMS and then use the test VM I use for webui (make vm-run) to test the real installer. 

I will select some other language than english, let's say de, and check for example this string. https://github.com/rhinstaller/anaconda-l10n/blob/master/master/de.po#L4208

If it's translated in plural I guess removing the semicolon works?

Reply:
Or maybe better we can add a unit test for it? Do you have unit tests for the translations?

Reply:
We have a unit test, but it tests the translation files, for things like missing formats etc. So not really in the way that is needed here.